### PR TITLE
[TAS-1042] 🚸 Save epub reader current page to localstorage

### PR DESCRIPTION
**Pull request by williamchong**

### CODE REVIEW

**Review by WeiJun0827 — APPROVED** [review]

File: src/pages/reader/epub.vue [hunk 1]
```diff
@@ -116,10 +123,16 @@ export default {
...
   methods: {
-    initRendition() {
+    async initRendition() {
+      this.isLoading = true;
       const encodedUrl = encodeURIComponent(this.fileSrc);
       const corsUrl = `https://pdf-cors-ufdrogmd2q-uw.a.run.app/pdf-cors?url=${encodedUrl}`;
-      this.book = Epub(corsUrl);
+      const buffer = await this.$axios.$get(corsUrl, {
+        responseType: 'arraybuffer',
+      });
```
Comment (nwingt) [inline src/pages/reader/epub.vue:132]: Should we add try-catch here?

Comment (williamchong) [reply src/pages/reader/epub.vue:132]: If any error is thrown here, the whole reader fail to function
Maybe we should show some error message

Comment (nwingt) [reply src/pages/reader/epub.vue:132]: It is acceptable if it shows an error page with the raw message

**Review by nwingt — APPROVED** [review]

File: src/pages/reader/epub.vue (hunk 1 above)
Comment (williamchong) [reply src/pages/reader/epub.vue]: added `this.$nuxt.error`

**Pull request merged**